src/main/main.ts

```diff
@@ -549,6 +561,7 @@ const createMainWindow = async () => {
     minHeight: WINDOW_HEIGHT,
     maxHeight: WINDOW_HEIGHT,
     icon: getAssetPath('icon.png'),
+    resizable: false,
```

Comment:
I added this because it prevents the "dragging the  window to the top of the screen causes Windows to try to resize it to fit the full screen" behavior.